pywinauto/windows/win32_hooks.py

```diff
@@ -67,10 +65,12 @@
 
 from .win32defines import VK_PACKET
 from pywinauto.actionlogger import ActionLogger
+from pywinauto.windows import win32functions
```

Comment:
We should use relative imports everywhere. Here: `from . import win32functions` and `from ..actionlogger import ActionLogger`